src/components/UncontrolledTabs.js

```diff
@@ -132,7 +133,7 @@ export default class UncontrolledTabs extends Component {
     }
 
     // Map children to dynamically setup refs
-    return React.Children.map(children, child => {
```

Comment:
This will not work when another react-tabs instance is nested inside I think, as it will find all children.
```js
<Tabs>
<TabPanel>
  <Tabs>
  <TabPanel></TabPanel>
  </Tabs>
</TabPanel>
</Tabs>
```

Not sure how to avoid this, as we also couldn't check if child === Tabs as children seem to be visited first.